content/terraform-docs-common/docs/registry/providers/docs.mdx

```diff
@@ -155,32 +157,40 @@ If you start a paragraph with a special arrow-like sigil, it will become a color
 
 ## Navigation Hierarchy
 
-Provider docs are organized by category: functions, ephemeral resources, resources, data sources, and guides. At a minimum, a provider must contain an index (`docs/index.md`) and at least one function, ephemeral resource, resource, or data source.
+Provider docs are organized by category: functions, actions, list resources, ephemeral resources, resources, data sources, and guides. At a minimum, a provider must contain an index (`docs/index.md`) and at least one function, action, ephemeral resource, list resource, resource, or data source.
```

Comment:
```suggestion
Provider docs are organized by the following categories: 

- functions 
- actions
- list resources
- ephemeral resources 
- resources 
- data sources 
- guides 

At a minimum, a provider must contain an index file, `docs/index.md`, and at least one function, action, ephemeral resource, list resource, resource, or data source.
```

Refer to the [style guide](https://github.com/hashicorp/web-unified-docs/blob/main/docs/style-guide/general/grammar.md#use-colons-to-introduce-lists-tables-and-visual-aids)